tests/test_resources_project.py

```diff
@@ -35,6 +36,62 @@ class UpdateTests(unittest.TestCase):
     def setUp(self):
         self.res = tower_cli.get_resource('project')
 
+    def test_create_with_organization(self):
+        """Establish that a project can be created inside of an organization
+        """
+        org_pk = 1
+        with client.test_mode as t:
+            endpoint = '/organizations/%s/projects/' % org_pk
+            t.register_json(endpoint, {'count': 0, 'results': [],
+                            'next': None, 'previous': None},
+                            method='GET')
+            t.register_json(endpoint, {'changed': True, 'id': 42},
+                            method='POST')
+            self.res.create(name='bar', organization=org_pk,
+                            scm_type="git")
+            self.assertEqual(t.requests[0].method, 'GET')
+            self.assertEqual(t.requests[1].method, 'POST')
+            self.assertEqual(len(t.requests), 2)
+
+    def test_create_without_organization(self):
+        """Establish that a project can be created without giving an org
+        """
+        with client.test_mode as t:
+            endpoint = '/projects/'
+            t.register_json(endpoint, {'count': 0, 'results': [],
+                            'next': None, 'previous': None},
+                            method='GET')
+            t.register_json(endpoint, {'changed': True, 'id': 42},
+                            method='POST')
+            self.res.create(name='bar', scm_type="git")
+            self.assertEqual(t.requests[0].method, 'GET')
+            self.assertEqual(t.requests[1].method, 'POST')
+            self.assertEqual(len(t.requests), 2)
+
+    def test_modify_organization_warning(self):
+        """Test that warning is thrown when trying to modify organization
+        because that can not be done in this way.
+        """
+        org_pk = 1
+        with mock.patch.object(warnings, 'warn') as warn:
+            with client.test_mode as t:
+                t.register_json('/projects/', {'count': 1, 'results': [{'id':1,
+                                'name':'bar'}],
+                                'next': None, 'previous': None},
+                                method='GET')
+                t.register_json('/projects/1/', {'name':'bar', 'id':1,
+                                'type':'project', 'organization':1},
+                                method='GET')
+                t.register_json('/projects/1/', {'name':'bar', 'id':1,
+                                'type':'project', 'organization':1},
+                                method='PATCH')
+                self.res.modify(name='bar', organization=org_pk,
+                                scm_type="git")
+                self.assertEqual(t.requests[0].method, 'GET')
+                self.assertEqual(t.requests[1].method, 'PATCH')
+                self.assertEqual(len(t.requests), 2)
```

Comment:
Above, you are registering 3 json endpoint httpd methods.  Here, you assert that only 2 are called.  Seems like we are have a mismatch?  Are lines 82-84 needed?  Does `tower-cli` issue a `GET` against `/projects/`, followed by a `GET` against `/projects/1/`, followed by a `PATCH` against `/projects/1/`?